.github/workflows/test_api_token.yml

```diff
@@ -22,12 +22,12 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install setuptools wheel twine
+        pip install build setuptools wheel twine
     - name: Build and publish
       env:
         TWINE_USERNAME: __token__
         TWINE_PASSWORD: ${{ secrets.TYPESHED_BOT_API_TOKEN }}
       run: |
         cd data/empty_package
-        python setup.py bdist_wheel ${{ github.event.inputs.increment }}
```

Comment:
What was this meant to do?
```
PS C:\Users\Avasam\AppData\Local\Temp\tmpk_pfo33j> python setup.py bdist_wheel 0
invalid command name '0'
```

Reply:
You can't upload wheels with the same version: https://github.com/typeshed-internal/stub_uploader/blob/main/data/empty_package/setup.py#L5

Reply:
Ah! I see how it's used now. Ok, I'll have to account for that. Setting as draft for now.